### ISSUE TITLE
Name property for Combo-box in 'Create a new Workbook' window is not defined 

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#338]
### Steps to Reproduce

1. Launch Xamarin Workbook application.
2. Navigate to 'Console' list item and select it
3. Navigate to Combo box and observe the Inspect property

### Expected Behavior
Name property for combo box should be properly defined 

### Actual Behavior
Name property for Combo box is not defined and currently is Empty

**Narrator behavior:** Narrator is not announcing  name of combo box and announcing name property of item selected in Combo box as 'Xamarin.Interactive.Client.ViewControllers.NewWorkbookItem, 4 of 4, selected'
**NVDA behavior:** NVDA is not announcing  name of combo box and announcing name property of item selected in Combo box as 'Xamarin.Interactive.Client.ViewControllers.NewWorkbookItem,'

### User Impact
Users who rely on screen reader will not be able to know the Name of combo box

![name_property_combo_box_inspect](https://user-images.githubusercontent.com/22956130/40722253-73121a92-6439-11e8-944d-964348601106.png)
![name_property_combo_box_narrator](https://user-images.githubusercontent.com/22956130/40722269-7e4e53a8-6439-11e8-92c5-53781fa28d43.png)
![name_property_combo_box_nvda](https://user-images.githubusercontent.com/22956130/40722283-87592acc-6439-11e8-9dea-fa72f2e6147b.png)

### Environment
Windows :Windows 10 Pro Version 1803 (OS Build : 17672.1000)
Xamarin Version : 1.4 (master/b3f92f9)
NVDA Version: 2018

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS4.2.1

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
